specs/TLA+/LeaderFollower/AbstractVeniceLeaderFollower.tla

```diff
@@ -0,0 +1,177 @@
+----------------------- MODULE AbstractVeniceLeaderFollower -----------------
+(***************************************************************************)
+(* This module describes the behavior of the Venice Leader follower model  *)
+(* in abstract terms.  Given at least one client writer, a venice leader   *)
+(* should be at some future state process all writes and persist them to a *)
+(* queue to be consumed by a set of follower nodes in such a way that all  *)
+(* all replicas become consistent at some state. The only concrete detail  *)
+(* we model here is that the transmission channel between a client  and    *)
+(* venice is that the channel is asynchronous and non destructive.         *)
+(***************************************************************************)
+
+EXTENDS Integers, Sequences
+CONSTANTS KEYS, VALUES, N_NODES
+
+(***************************************************************************)
+(* Variables which define the spec and parts of the architecture:          *)
+(*                                                                         *)
+(* realTimeTopic: A queue of writes transmitted to Venice directly from    *)
```

Comment:
Do we want to highlight this spec is for one single partition since each topic could contain multiple partitions?

Reply:
Sure